### PR TITLE
981-fe-refactor-avatar-props 

### PR DESCRIPTION
**Pull request by REALcuba**
Refactor avatar props so it can be use as userbtn.

### CODE REVIEW

File: packages/ui/src/components/atoms/Avatar/index.tsx [hunk 1]
```diff
@@ -1,30 +1,56 @@
...
+  mediaQuery = { ...defaultMediaQuery },
+  onClick
+}) => (
+  <StyledAvatar
+    src={src}
+    alt={alt}
+    avatarCss={avatarCss}
+    ref={forwardedRef}
+    mediaQuery={mediaQuery}
+    onClick={onClick}
```
Comment (NataliaR-BCN) [inline packages/ui/src/components/atoms/Avatar/index.tsx:54]: What's mediaQuery prop for? It seems to repeat/contradict avatarCss ¿?

File: packages/ui/src/__tests__/atoms/Avatar.test.tsx [hunk 2]
```diff
@@ -1,13 +1,27 @@
...
     expect(image).toBeInTheDocument()
     expect(image).toHaveAttribute('src', '/test.png')
     expect(image).toHaveAttribute('alt', 'users portrait')
+   
+    expect(image).toHaveStyle('width: 97px; height: 97px; border-radius: 50%;');
+
+    fireEvent.click(image)
+    expect(mockClick).toHaveBeenCalled()
+
   })
```
Comment (NataliaR-BCN) [inline packages/ui/src/__tests__/atoms/Avatar.test.tsx:26]: This test is ok, but also include one first with the component by default, without optional props, please.

File: packages/ui/src/components/atoms/Avatar/index.tsx (hunk 1 above)
Comment (REALcuba) [reply packages/ui/src/components/atoms/Avatar/index.tsx:54]: mediaquery are the variable that will be used when passing diferents @media depending in where is going to be used, for navbar need some media style and card profiles a diferent one, I took them from the original file, now that userbtn will be removed.

File: packages/ui/src/__tests__/atoms/Avatar.test.tsx (hunk 2 above)
Comment (REALcuba) [reply packages/ui/src/__tests__/atoms/Avatar.test.tsx:37]: Test without passing css props, done, thansk

**Review by marmorrei — APPROVED** [review]

**Review by NataliaR-BCN — APPROVED** [review]

**Pull request merged**